install/Knative-with-GKE.md

```diff
@@ -59,7 +61,8 @@ You need a GCP project to create a Google Kubernetes Engine cluster.
    existing GCP project as your `gcloud` default:
     * If you don't already have a GCP project created, create a new project in `gcloud`:
       ```bash
-      gcloud projects create my-knative-project --set-as-default
+      export PROJECT=my-knative-project
```

Comment:
This would be better set above this step, so it can apply to the "already have a project" step.

Reply:
hm, I had a few versions yesterday, but of course you're right.